assets/_scss/components/_forms.scss

```diff
@@ -0,0 +1,17 @@
+input.input-small {
+  @include media($tablet-up) {
+    max-width: 6rem;
+  }
+}
+
+.additional_text {
+  float: right;
+  font-style: italic;
+  font-weight: normal;
+}
+
+.usa-form-one-third {
```

Comment:
Bc other forms have a different width like one half on the address block. What do you think we should do for this issue?